### ISSUE TITLE
creating a trawmeta tree

### ISSUE THREAD

**Issue by mjtueros** [jelenapetereit/GrandRawRoot#7]
i propose to put meta information (information that is not part of the input of zhaires/coreas, like unix time or core position)
in a separate class. The reason for this is to have the posibilitiy to create and fill this class at the time of input generation if desired, with  a separate function for this that would be independent of ZHAireS and COREAS, and thus could be maintained independently (and only once).
This would imply moving core_position and unix from rawshowertree to rawmetatree (that i am proposing to create)
